### PR TITLE
os/tools: Validate packages after build

### PR DESCRIPTION
**Pull request by jeongarmy**
Call script for checking package validation after build.
It parses the package header information and show them.
And verify the package by calculating the crc and comparing it with the value of the header.

### CODE REVIEW

**Comment by sunghan-chang** [comment]
Which kind of validation is included? Could you leave what it is in the commit description?

File: os/tools/check_output.py [hunk 1]
```diff
@@ -61,6 +62,16 @@ def check_binary_size(bin_type, part_size):
         os.remove(output_path)
         FAIL_TO_BUILD = True
 
+def check_binary_header(bin_type, part_size):
```
Comment (abhishek-samsung) [inline os/tools/check_output.py:65]: Is part_size required in this function?

File: os/tools/check_output.py [hunk 2]
```diff
@@ -133,4 +144,14 @@ def check_binary_size(bin_type, part_size):
...
+
+# Validate the header data of binaries
+check_binary_header("KERNEL", KERNEL_PARTITION_SIZE)
+if CONFIG_APP_BINARY_SEPARATION == "y" :
+    if APP1_IDX != 0 :
+        check_binary_header("APP1", APP1_PARTITION_SIZE)
+    if APP2_IDX != 0 :
+        check_binary_header("APP2", APP2_PARTITION_SIZE)
+    if CONFIG_SUPPORT_COMMON_BINARY == "y" :
+        check_binary_header("COMMON", COMMON_PARTITION_SIZE)
```
Comment (sunghan-chang) [inline os/tools/check_output.py:157]: Could you check the line end?

**Review by sunghan-chang — COMMENTED** [review]
I can't distinguish purpose of each, check_output.py and check_package.py.
`output` and `package` looks the same.
Could you rename them to know what it is from the name?

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**